content/docs/courses/zero-fundamentals/tcp-routes.mdx

```diff
@@ -0,0 +1,198 @@
+  -v, --version   version for pomerium-cli
+
+Use "pomerium-cli [command] --help" for more information about a command.
+```
+
+## Add Redis and OpenSSH services
+
+Add the Redis and OpenSSH server configurations to your Docker Compose file:
+
+```yaml
+redis:
+  image: redis:latest
+  networks:
+    main: {}
+  expose:
+    - 6379
+
+myssh:
+  image: linuxserver/openssh-server:latest
+  networks:
+    main: {}
+  expose:
+    - 2222
+  environment:
+    PASSWORD_ACCESS: 'true'
+    USER_PASSWORD: supersecret
+    USER_NAME: user
+```
+
+## Add routes in Pomerium Zero
+
+Create a Redis route:
+
+1. Create a new route for your Redis database
+1. In **From**, select **tcp+https://** in the protocol dropdown menu
+1. Enter your external route and append `:6379` to it
+   1. For example, `redis.super-hero-7645.pomerium.app:6379`
+1. In **To**, enter `tcp://redis:6379`
+1. In **Policies**, select `Any Authenticated User`
+
+   ![Building the Redis route in the Zero Console](./img/tcp/redis-service.png)
+
+Create an OpenSSH route:
+
+1. Create a new route for the OpenSSH server
+1. In **From**, select **tcp+https://** in the protocol dropdown menu
+1. Enter your external route and append `:22` to it
+   1. For example, `myssh.super-hero-7645.pomerium.app:22`
+1. In **To**, enter `tcp://myssh:2222`
+
+   ![Building the OpenSSH route in the Zero Console](./img/tcp/ssh-server.png)
+
+Save your changes and apply them.
+
+## Connect to Redis
+
+In a terminal, run the following command:
+
+```shell-session
+$ pomerium-cli tcp redis.<CLUSTER_SUBDOMAIN>.pomerium.app:6379 --listen localhost:6379
+```
+
+If set up correctly, Pomerium will open your browser to authenticate you. This will establish a TCP connection.
+
+In a separate terminal window, run the following `redis-cli` command:
+
+```bash
+$  redis-cli info
+# Server
+redis_version:7.0.5
+redis_git_sha1:00000000
+redis_git_dirty:0
+redis_build_id:d9291579292e26e3
+redis_mode:standalone
+os:Linux 6.3.13-linuxkit aarch64
+arch_bits:64
+monotonic_clock:POSIX clock_gettime
+multiplexing_api:epoll
+atomicvar_api:c11-builtin
+gcc_version:10.2.1
+process_id:1
+process_supervised:no
+run_id:bc1b8bcd39f1e51d615f5739158e6ae964f7e724
+tcp_port:6379
+server_time_usec:1713989553900448
+uptime_in_seconds:64
+uptime_in_days:0
+hz:10
+configured_hz:10
+lru_clock:2713521
+executable:/data/redis-server
+config_file:
+io_threads_active:0
+```
+
+We truncated the Redis response above for the sake of brevity, but it demonstrates that you successfully proxied a TCP connection to the Redis service.
+
+## Connect to SSH server
+
+In a terminal, run the following command:
+
+```shell-session
+$ pomerium-cli tcp myssh.<CLUSTER_SUBDOMAIN>.pomerium.app:22 --listen :2222
+```
+
+If set up correctly, Pomerium will open your browser to authenticate you. This will establish a TCP connection.
+
+In a separate terminal window, run:
+
+```bash
+ssh user@myssh.<CLUSTER_SUBDOMAIN>.pomerium.app -p 2222
```

Comment:
I think this needs to be sent to localhost, since the cli is facilitating the connection.  

```suggestion
ssh user@localhost -p 2222
```